blog/app/views/articles/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>
+	<strong>Title:</strong>
+  	<%= @article.title %>
+</p>
+<p>
+	<strong>Test:</strong>
+	<%= @article.text %>
```

Comment:
Get used to use the function `simple_format` on view having a text field. 
http://apidock.com/rails/ActionView/Helpers/TextHelper/simple_format

_database textfield != string_